python_scripts/shellies_discovery_gen2.py

```diff
@@ -933,7 +938,17 @@
     KEY_JSON_ATTRIBUTES_TOPIC: TOPIC_STATUS_SYS,
     KEY_JSON_ATTRIBUTES_TEMPLATE: TPL_FIRMWARE_STABLE_ATTRS_INDEPENDENT,
 }
-DESCRIPTION_SLEEPING_SENSOR_HUMIDITY = {
+DESCRIPTION_SENSOR_HUMIDITY = {
+    KEY_DEVICE_CLASS: DEVICE_CLASS_ILLUMINANCE,
```

Comment:
This appears to be backwards, so humidity on my Plus H&T shows as illuminance and I suppose on other devices vice versa.

Reply:
Please open an issue.

Reply:
Made https://github.com/bieniu/ha-shellies-discovery-gen2/pull/323 instead.